packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -57,6 +57,8 @@ export const useSliderState = (state: Pick<SliderState, keyof SliderCommon | key
   const onChangeCallback = React.useRef(onChange);
   const id = useId('slider-', state.id);
 
+  onChangeCallback.current = onChange;
+
```

Comment:
```suggestion
```

Please consider to use `useEventCallback()` instead of this:

```diff
-const onChangeCallback = React.useRef(onChange);
```

```diff
-  const updateValue = React.useCallback(
+  const updateValue = useEventCallback(
```

```diff
-      if (onChangeCallback.current) {
-        onChangeCallback.current(ev, { value: clampedValue });
-      }
+      if (onChange) {
+        onChange(ev, { value: clampedValue });
+     }
```

```diff
-[max, min, setCurrentValue],
```